### PR TITLE
[4.0] Point to J4 translations list

### PR DESCRIPTION
**Pull request by wilsonge**
Pull Request for Issue #21806.

### Summary of Changes
This sets up the language files in J4 to point to a J4 language server.

NOTE: This file doesn't exist yet - it will in the next 24 hours. this should NOT be merged until that file exists

### Testing Instructions
Ensure Joomla is updated to point to the new update list.

### Documentation Changes Required
None

### CODE REVIEW

File: administrator/components/com_admin/sql/updates/mysql/4.0.0-2021-05-30.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+UPDATE `#__update_sites`
+   SET `location` = 'https://update.joomla.org/language/translationlist_4.xml'
+ WHERE `update_site_id` = 2;
```
Comment (richard67) [inline administrator/components/com_admin/sql/updates/mysql/4.0.0-2021-05-30.sql]: You could have added that to the existing 4.0.0-2021-05-21.sql script as we don't support updating from nightlies.

File: administrator/components/com_admin/sql/updates/postgresql/4.0.0-2021-05-30.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+UPDATE "#__update_sites"
+   SET "location" = 'https://update.joomla.org/language/translationlist_4.xml'
+ WHERE "update_site_id" = 2;
```
Comment (richard67) [inline administrator/components/com_admin/sql/updates/postgresql/4.0.0-2021-05-30.sql]: You could have added that to the existing 4.0.0-2021-05-21.sql script as we don't support updating from nightlies.

File: administrator/components/com_admin/sql/updates/mysql/4.0.0-2021-05-30.sql (hunk 1 above)
Comment (richard67) [reply administrator/components/com_admin/sql/updates/mysql/4.0.0-2021-05-30.sql]: And maybe renamed it and add a comment to it with the new script name like we have the other comments after we combined the SQL.

But that's just a nice to have, the PR here is good as it is.

File: administrator/components/com_admin/sql/updates/postgresql/4.0.0-2021-05-30.sql (hunk 2 above)
Comment (richard67) [reply administrator/components/com_admin/sql/updates/postgresql/4.0.0-2021-05-30.sql]: See also additional comment in my other review comment above.

**Comment by wilsonge** [comment]
@richard67 any chance you can PR to my branch. I've completely destroyed my PHP setup last night trying to update some homebrew stuff :(

**Comment by richard67** [comment]
For the update SQL, can we be sure that the ID of the update site which we have to change is always 2, also on installation with a longer update history?

**Comment by richard67** [comment]
On my home page the ID is 3.

**Comment by alikon** [comment]
use the `location` in the where condition instead of `update_site_id`

**Comment by richard67** [comment]
@alikon You are right. I've just fixed that.

**Pull request merged**